articles/container-registry/container-registry-faq.md

```diff
@@ -407,6 +408,17 @@ curl $redirect_url
 
 If you are using the Microsoft Edge/IE browser, you can see at most 100 repositories or tags. If your registry has more than 100 repositories or tags, we recommend that you use either the Firefox or Chrome browser to list them all.
 
+### Why is Azure portal unable to send request for fetching repositories or tags?
+
```

Comment:
Suggest rewording heading to something like "Why don't my repositories or tags appear in the Azure portal?" - assuming that's the customer's experience of the problem. Thanks.

Reply:
@dlepow the question "Why don't my repositories or tags appear in the Azure portal?" is too generic and could have a myriad of reasons or answers.

We are trying to address a specific problem here, i.e. the portal is unable to send requests to the server due to status zero errors (status zero being internal). Perhaps, the question could be rephrased while maintaining the specificity and context. Any thoughts?

cc // @sajayantony @ankurkhemani

Reply:
Maybe we can do: 
`Why does Azure portal fail to fetch repositories or tags`?

Reply:
@dlepow @ankurkhemani 
Question rephrased to `Why does the Azure portal fail to fetch repositories or tags?`